src/Transition/CreateNewIDFUsingRulesV8_6_0.f90

```diff
@@ -478,7 +478,16 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                 OutArgs(3)=InArgs(5)
                 CurArgs = CurArgs-2
 
-              CASE('BRANCH')
+            CASE('AIRTERMINAL:SINGLEDUCT:VAV:REHEAT')
+                nodiff=.false.
+                OutArgs=InArgs
+                IF (SameString(InArgs(16), 'REVERSE')) THEN
+                  IF ( (.NOT. SameString(InArgs(17),'')) .OR. (.NOT. SameString(InArgs(18),''))  ) THEN
+                    OutArgs(16)='ReverseWithLimits'
+                  END IF
+                END IF
+
```

Comment:
@wfbuhl (@mjwitte) Is this correct?  I followed your Rules file changes.  Basically 

 - if F16 is "REVERSE" then we do stuff:
   - if F17 is NOT BLANK; or if F18 is NOT BLANK then:
     - Change it to ReverseWIthLimits
 - Otherwise just leave it.

Note, when I test-transitioned a file, it changed `Reverse` to `ReverseWithLimits` because F17 and F18 were `AutoCalculate`.  That's good, right?